cli/submit.py

```diff
@@ -15,9 +18,11 @@
 
 
 class StudySubmitter:
-    def __init__(self, submission_initiate_url=SUBMISSION_INITIATE_URL):
+    def __init__(self, vcf_files, metadata_file, submission_initiate_url=SUBMISSION_INITIATE_URL):
```

Comment:
Should change [this initialisation](https://github.com/EBIvariation/eva-sub-cli/blob/17a351a25f2ad436344d16e58cbd7c40ca79ca7d/cli/eva_sub_cli.py#L16) as well... the user interaction might be a bit rough but we should get something roughly working first.

Reply:
This was changing significantly in the next [PR](https://github.com/EBIvariation/eva-sub-cli/pull/14/files#diff-2eec00526c12fe9eab725e922d6e5fb4520e7769add0871942ddc1df8cedd5e4) which is why I left it as is

Reply:
Got it, I see why I was confused..